roles/grafana/defaults/main.yml

```diff
@@ -145,6 +145,9 @@ grafana_ldap: {}
 #        - group_dn: "cn=alternative_admins,ou=groups,dc=grafana,dc=org"
 #          org_role: Admin
 
+# Grafana KeyCloak auth
+grafana_auth_generic_oauth: {}
```

Comment:
Would be nice if you can add a small example int the form of comments.
grafana_session on line 148 can serve as an example.

Reply:
Sure, I would.

Reply:
The example you requested is added. :)